gybitg/Views/Manage/ManageNavPages.cs

```diff
@@ -25,15 +25,18 @@ public static class ManageNavPages
 
         public static string AthleteList => "AthleteList";
 
+        //added by Kevin on 4/15/19
+        public static string BasicSearch => "BasicSearch";
+
         public static string SearchResults => "SearchResults";
 
         public static string AdvancedSearch => "AdvancedSearch";
 
         public static string TwoFactorAuthentication => "TwoFactorAuthentication";
 
-        public static string IndexNavClass(ViewContext viewContext) => PageNavClass(viewContext, Index);
+        //public static string BasicSearchClass(ViewContext viewContext) => BasicSearchClass(viewContext, BasicSearch);
 
-        public static string AdvancedSearchNavClass(ViewContext viewcontext) => PageNavClass(viewcontext, AdvancedSearch);
```

Comment:
Did you get Alex's permission to delete his AdvancedSearchNavClass and why is yours commented out?

Reply:
Not sure.  I didn't delete anyone's code.  And I didn't comment out any of my own code lines.

Reply:
Only thing I deleted was my own BasicSearchViewModel.cs

Reply:
Could you add it back in and uncomment your NavClass before merging it.